_posts/2016-12-x-kinetic-release.markdown

```diff
@@ -0,0 +1,44 @@
+---
+author: 130s
+comments: false
+date: 2016-12-x 00:00:00+00:00
+layout: post
+slug: firstkineticrelease
+title: MoveIt! officially released into ROS Kinetic
+media_type: image
+media_link: http://wiki.ros.org/kinetic?action=AttachFile&do=get&target=kinetic.png
+description: If you're running your robot packages with ROS Kinetic, and/or if you want to try the newest MoveIt!, start at moveit.ros.org/install
+categories:
+- MoveIt!
+- ROS
+---
+
+MoveIt! finally made it to the latest ROS LTS (Long Term Support) version [Kinetic](http://wiki.ros.org/kinetic)!
+From now onward, Kinetic is and will be the ROS version that comes with the newest capabilities and bug fixes for MoveIt!.
+
+First thing to note is that from Kinetic C++11 is fully enforced for MoveIt!, which means you can NOT build its source branch ([kinetic-devel](https://github.com/ros-planning/moveit/tree/kinetic-devel)) on the computers where the previous ROS distros run.
+
+Some new features and bug-fixes added since the [last release into Jade](http://moveit.ros.org/moveit!/ros/2016/09/02/firstjaderelease.html), :
+
+- [Trajectory safety validation before execution](https://github.com/ros-planning/moveit/pull/63) (Kinetic onward only)
+- New planners in [OMPL](http://ompl.kavrakilab.org/) are now available from MoveIt! (geometric::{FMT, BFMT, PDST, STRIDE, BiTRRT, LBTRRT, BiEST, ProjEST, LazyPRM, LazyPRMstar, SPARS, SPARStwo}) ([see PR for detail](https://github.com/ros-planning/moveit/pull/338))
+
+- [MoveGroup's default capabilities are now added by default](https://github.com/ros-planning/moveit/pull/359).
+- [New benchmarking using PlanningArena](https://github.com/ros-planning/moveit/pull/228)
+- [Fix falsely called grasp service](https://github.com/ros-planning/moveit/pull/328)
+- [Increase collision checking reliability for newly created moveit_config packages](https://github.com/ros-planning/moveit/pull/337)
+- [IK solvers consolidated into a new moveit_kinematics package](https://github.com/ros-planning/moveit/pull/247)
+- [Fix unreasonable "Invalid Trajectory: start point deviates" error](https://github.com/ros-planning/moveit/issues/283)
+- Fedora support is added. Feedback (and fix if necessary) welcomed!
+
+If you're running your robot packages with ROS Kinetic, and/or if you want to try the newest MoveIt!, start at [moveit.ros.org/install](http://moveit.ros.org/install/)
+
+Update on ROS Indigo 
+========================
+
+Contributors are working hard towards a new binary release into Indigo as well. Our loosely enforced backport policy is as follows. That said, Kinetic is recommended if you'd like to utilize the result of newest development:
```

Comment:
@davetcoleman probably agrees.
I don't see the need to reduce indigo support given that numerous robots out in the wild will still run indigo next year's christmas. How about:

> - Future research will primarily target Kinetic. ABI-compatible changes can be released into previous ROS distros too.

I'd propose to drop the point about bug-fixes. Bug-fixes should always be merged and released into all supported distributions.